client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -266,10 +269,12 @@ private void submitRetryPushMergedData(
         if (remainReviveTimes > 0) {
           reviveFailedBatchesMap.add(batch);
         } else {
-          pushState.exception.compareAndSet(
-              null,
-              new CelebornIOException(
-                  "Revive Failed in retry push merged data for location: " + batch.loc));
+          String errorMsg =
+              String.format(
+                  "Revive failed while pushing merged for shuffle %d map %d attempt %d partition %d batch %d location %s.",
+                  shuffleId, mapId, attemptId, partitionId, oldGroupedBatchId, batch.loc);
+          logger.debug(errorMsg);
+          pushState.exception.compareAndSet(null, new CelebornIOException(errorMsg));
```

Comment:
When pushState.exception is thrown, the errorMsg will be print, so will it be duplicate?

Reply:
> When pushState.exception is thrown, the errorMsg will be print, so will it be duplicate?

Yea, don't need to print the error.